src/ts/components/errormessageoverlay.ts

```diff
@@ -2,8 +2,12 @@ import {ContainerConfig, Container} from './container';
 import {Label, LabelConfig} from './label';
 import {UIInstanceManager} from '../uimanager';
 import {TvNoiseCanvas} from './tvnoisecanvas';
-import {ErrorUtils} from '../errorutils';
+import {ErrorUtils } from '../errorutils';
```

Comment:
Either before and after or no space 😄 
```suggestion
import { ErrorUtils } from '../errorutils';
```